README.md

```diff
@@ -17,19 +17,26 @@ This means it's not possible to use anything in the [`std`](https://doc.rust-lan
 
 ## SDK version
 
-Currently supports SDK 20.0 ([flipperzero-firmware@0.80.1](https://github.com/flipperdevices/flipperzero-firmware/tree/0.80.1)).
+Currently supports SDK 26.0 ([flipperzero-firmware@0.x.x](https://github.com/flipperdevices/flipperzero-firmware/tree/dev)).
 
 The crate major version number will be updated after a bump in [API version](https://github.com/flipperdevices/flipperzero-firmware/blob/release/firmware/targets/f7/api_symbols.csv) in the Flipper Zero firmware.
 
 | Crate version | API version |
 | ------------- | ----------- |
+| 0.x.x         | 26.0        |
 | 0.8.x         | 20.0        |
 | 0.7.x         | 14.0        |
 | 0.6.x         | 11.2        |
 | 0.5.x         | 10.1        |
 | 0.4.x         | 7.5         |
 | 0.3.x         | 2.2         |
 
+To build a newer version of the bindings:
+
+```shell
+docker run --rm -t $(docker build -q .) > crates/sys/src/bindings.rs
+```
+
```

Comment:
Instead of putting this content here, please amend `docs/updating-sdk.md` to describe the Docker flow alongside the non-Docker flow.

Reply:
make sense